packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionTncSection/JurisdictionTncSection.scss

```diff
@@ -24,6 +24,11 @@
         gap: 0.8rem;
         justify-content: center;
 
+        &-input {
+            all: revert;
+            accent-color: #ff444f;
+        }
```

Comment:
Added css properties to overwrite the deriv-com/ui properties below which was added to remove default behaviour for some component types:

<img width="417" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/d4b3a003-1fd5-433d-905e-0d306aa5dc4a">